src/components/common/copyable-button/copyable-button.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Fragment, useCallback, useRef, useState } from 'react'
+import { Button, Overlay, Tooltip } from 'react-bootstrap'
+import { Variant } from 'react-bootstrap/types'
+import { Trans, useTranslation } from 'react-i18next'
+import { ForkAwesomeIcon } from '../fork-awesome/fork-awesome-icon'
+
+export interface CopyableButtonProps {
+  content: string
+  size?: 'sm' | 'lg'
+  variant?: Variant
+}
+
+export const CopyableButton: React.FC<CopyableButtonProps> = ({ content, size = 'sm', variant = 'dark' }) => {
+  const { t } = useTranslation()
+  const button = useRef<HTMLButtonElement>(null)
+  const [showCopiedTooltip, setShowCopiedTooltip] = useState(false)
+
+  const copyToClipboard = useCallback((content: string) => {
+    navigator.clipboard.writeText(content).then(() => {
+      setShowCopiedTooltip(true)
+      setTimeout(() => { setShowCopiedTooltip(false) }, 2000)
+    }).catch(() => {
+      console.error("couldn't copy")
+    })
+  }, [])
+
+  return (
+    <Fragment>
+      <Overlay target={button} show={showCopiedTooltip} placement="top">
+        {(props) => (
+          <Tooltip id={'copied_' + content} {...props}>
```

Comment:
why are you adding an id?

Reply:
that's copied from CopyableField

Reply:
If you don't need the id, then please remove it.

Reply:
Not really sure. Why the I'd @ErikMichelson ?

Reply:
Okay it seems like the tooltip component needs this id.